feat: add 2023-11-16 (#37)
OmnicodeSolutions/worklog-luisa: content/blog/2023-11-16.md

+++
title = "2023-11-16"
date = 2023-11-16
+++

---

Today is Thursday, day 320 of 365 (87.67%). We're in week 46 of 52 (88.46%).

## Planning

What I am going to do today:

- [ ] finish unit tests for the [Flutter Client](https://github.com/OmnicodeSolutions/luisa_drf_flutter_client)
- [ ] work on [issue#1](https://github.com/OmnicodeSolutions/luisa_drf_flutter_client/issues/1) from [Flutter Client](https://github.com/OmnicodeSolutions/luisa_drf_flutter_client)

## Activities executed

Today I dedicated the whole morning to try finishing the unit tests but I am still having some trouble with it, however I have found the issue and hope to fix it tomorrow, I believe I am trying to find something that has not yet been created, since the app uses a [Future builder](https://api.flutter.dev/flutter/widgets/FutureBuilder-class.html).

## Learnings

> Today I learned Dart's library [nock](https://pub.dev/packages/nock), which is used for test modules that perform http requests, since those request are not acctually made during tests.

Reviewer: ```suggestion
Today I learned Dart's library [nock](https://pub.dev/packages/nock), which is used for test modules that perform http requests, since those requests should not actually be made during tests.
```
